adr/0005-focus-group-process.adoc

```diff
@@ -0,0 +1,160 @@
+= Focus Groups
+
+* Status: _proposed_
+* Date: 2024-07-01 by @cescoffier
+
+== Context and Problem Statement
+
+Quarkus is a large project with many contributors.
+It's hard to keep track of all the initiatives and ensure that the community is aware of ongoing work.
+We need a way to organize work around specific topics and ensure that the community is aware of these initiatives.
+
+We also need to ensure that the work is done transparently and that the community can participate in the discussions.
+
+This would be the basis of an informal roadmap, where the community can see what is being worked on and what is coming next.
+Our previous attempts to publish and maintain a roadmap were not successful.
+We need a more lightweight approach, focusing on current work and next steps.
+
+Additionally, new contributors may find it hard to find a way to contribute to the project, as the project's size may be overwhelming.
+Working on a specific area or topic may be more appealing to new contributors and may help them get started.
+
+== Focus Groups
+
+The idea behind this proposal is to introduce the concept of _focus groups_.
+A focus group is a lightweight way to organize work around a specific topic.
+It aims to gather people interested in a specific topic and ensure that the work is done transparently.
+It also aims to ensure that the community is aware of ongoing work and can participate in the discussions.
+
+=== Defining a Focus Group
+
+To initiate a focus group, we need to define the scope of the focus group.
+This should be a specific topic or initiative that needs to be worked on.
+The definition of a focus group should follow the _SMART_ criteria:
```

Comment:
been thinking on this mention of SMART (which btw. for me is not exactly equal to "light weight":) 

are you suggesting these gets called out in the discussion or just a good goal? 

I can do most of them fairly easily except measurable...are you just asking for it being trackable ? as measurable from SMART is normally "15% faster than X" or some other quantity not just "when all issues are resolved".

Reply:
I don't like mentioning SMART either. I just needed a way to explain how to scope a working group, to avoid the: let's rebuild the universe. 

Let me rephrase it a bit.

Reply:
Done. LTM.